### ISSUE TITLE
Strange issue on Lakka

### ISSUE THREAD

**Issue by CometHunter92** [libretro/pcsx_rearmed#828]
### PCSX-ReARMed Version

r24l 87a0a6318564fc171d0b576da8ea7e629aa229e7

### Your device

Pi4

### Operating System of your device

Other (consoles, etc.)

### CPU architecture

ARM 64bit (AArch64)

### Issue description

I know it's ok to left the "Region" option to "auto", but in my Lakka device I have already individual .opt files for every game from 2 years now, and many PAL games were saved with the "PAL" region specified in the option file. Well, something strange happened after one of the latest commits: those games stopped working, crashing at the core loading. 
Basically if "PAL" is specified in the .opt file, the game doesn't start, crashing the core. If I remove the .opt files, or change the "region" value to "auto" they work as before. It only happens when "PAL" is selected, weirdly, games with "NTSC" values in the .opt files work. Didn't test on other platforms at the moment.
Edit:
when on "auto" PAL games run at NTSC speed. So something wrong is happening there.

The issue started at https://github.com/libretro/pcsx_rearmed/commit/2f326fa15ff26df057ef10da232ebc07a410803a

**Comment by CometHunter92** [comment]
Tried on RG35XX too, the PAL games start but no audio is reproduced.
